Student-Grace-Calculation/src/GradeSchemaCalculator.java

```diff
@@ -35,6 +35,11 @@ public static void main(String[] args) {
 		
 		Scanner scan = new Scanner(System.in);
 		
+		String showResults = "Your provided details for Grade Schema are like: \n"+
```

Comment:
a bit above you have declaration of  String[] grades. Can you make the values: "A+", "A", "B+", "B", "C+", "C", "D+", "D", "F" to be constant somehow. You can use enum for this, or just create a static constants in the GradeInfo class. The reason for this is that we most likely will use these later in the code as well. A reason for moving this data is that driver class will be removed later so we should keep data that affects our business logic in another file so it will be easy to delete the driver class.